en/tutorial-base.md

```diff
@@ -258,13 +286,15 @@ try {
     echo 'Exception: ', $e->getMessage();
 }
 ```
-
-As you can see, the bootstrap file is very short and we do not need to include any additional files. We have set ourselves a flexible MVC application in less than 30 lines of code.
+<br/>
+As you can see, the bootstrap file is very short and we do not need to include any additional files. **Congratulations** you are well on your to having created a flexible MVC application in less than 30 lines of code.
 
 <a name='controller'></a>
 ## Creating a Controller
 By default Phalcon will look for a controller named "Index". It is the starting point when no controller or action has been passed in the request. The index controller (`app/controllers/IndexController.php`) looks like:
```

Comment:
Generally, `this` is meaningless with out referring to a direct object. While stating it with `this` sounds more informal / naturally conversational I would prefer say
`would look like the following:`
OR
`should resemble the example below:`